### PR TITLE
DOCSUP-11554: Edit and translate to Russian

### PR DESCRIPTION
**Pull request by ka1bi4**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #26055 and #26130.

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

- Added ClickHouse dictionary source secure setting documentation.
- Functions dictGet, dictHas complex key dictionary key argument tuple fix.

### CODE REVIEW

File: docs/ru/sql-reference/functions/ext-dict-functions.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ dictHas('dict_name', id)
 **Аргументы**
 
 -   `dict_name` — имя словаря. [Строковый литерал](../syntax.md#syntax-string-literal).
--   `id_expr` — значение ключа словаря. [Выражение](../syntax.md#syntax-expressions), возвращающее значение типа [UInt64](../../sql-reference/functions/ext-dict-functions.md) или [Tuple](../../sql-reference/functions/ext-dict-functions.md) в зависимости от конфигурации словаря.
+-   `id_expr` — значение ключа словаря.  [Выражение](../../sql-reference/syntax.md#syntax-expressions) возвращает значение словаря ключ-значение или [Tuple](../../sql-reference/functions/ext-dict-functions.md), в зависимости от конфигурации словаря.
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/ext-dict-functions.md:240]:
```suggestion
-   `id_expr` — значение ключа словаря.  [Expression](../../sql-reference/syntax.md#syntax-expressions) возвращает пару "ключ-значение" словаря или [Tuple](../../sql-reference/functions/ext-dict-functions.md) в зависимости от конфигурации словаря.
```

Comment (adevyatova) [reply docs/ru/sql-reference/functions/ext-dict-functions.md:240]: `Expression` here is a data type name, it shouldn't translate.

File: docs/ru/sql-reference/functions/ext-dict-functions.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ dictGetOrNull('dict_name', attr_name, id_expr)
 
 -   `dict_name` — имя словаря. [Строковый литерал](../syntax.md#syntax-string-literal).
 -   `attr_names` — имя столбца словаря, [Строковый литерал](../syntax.md#syntax-string-literal), или кортеж [Tuple](../../sql-reference/data-types/tuple.md) таких имен.
--   `id_expr` — значение ключа словаря. [Выражение](../syntax.md#syntax-expressions), возвращающее значение типа [UInt64](../../sql-reference/functions/ext-dict-functions.md) или [Tuple](../../sql-reference/functions/ext-dict-functions.md), в зависимости от конфигурации словаря.
+-   `id_expr` — значение ключа словаря.  [Выражение](../../sql-reference/syntax.md#syntax-expressions) возвращает значение словаря ключ-значение или [Tuple](../../sql-reference/functions/ext-dict-functions.md), в зависимости от конфигурации словаря.
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/ext-dict-functions.md:27]:
```suggestion
-   `id_expr` — значение ключа словаря.  [Expression](../../sql-reference/syntax.md#syntax-expressions) возвращает пару "ключ-значение" словаря или [Tuple](../../sql-reference/functions/ext-dict-functions.md), в зависимости от конфигурации словаря.
```

File: docs/ru/sql-reference/functions/ext-dict-functions.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ dictGetOrNull('dict_name', attr_name, id_expr)
 
 -   `dict_name` — имя словаря. [Строковый литерал](../syntax.md#syntax-string-literal).
 -   `attr_names` — имя столбца словаря, [Строковый литерал](../syntax.md#syntax-string-literal), или кортеж [Tuple](../../sql-reference/data-types/tuple.md) таких имен.
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/ext-dict-functions.md:26]:
```suggestion
-   `attr_names` — имя столбца словаря. [Строковый литерал](../syntax.md#syntax-string-literal), или кортеж [Tuple](../../sql-reference/data-types/tuple.md) таких имен.
```

File: docs/ru/sql-reference/dictionaries/external-dictionaries/external-dicts-dict-sources.md [hunk 4]
```diff
@@ -609,6 +611,7 @@ SOURCE(CLICKHOUSE(
 -   `table` — имя таблицы.
 -   `where` — условие выбора. Может отсутствовать.
 -   `invalidate_query` — запрос для проверки статуса словаря. Необязательный параметр. Читайте подробнее в разделе [Обновление словарей](external-dicts-dict-lifetime.md).
+-   `secure` - флаг, разрешающий или не разрешающий защищенное SSL-соединение.
```
Comment (mathalex) [inline docs/ru/sql-reference/dictionaries/external-dictionaries/external-dicts-dict-sources.md:614]:
```suggestion
-   `secure` - флаг, разрешающий или не разрешающий защищённое SSL-соединение.
```

File: docs/ru/sql-reference/functions/ext-dict-functions.md [hunk 5]
```diff
@@ -23,8 +23,8 @@ dictGetOrNull('dict_name', attr_name, id_expr)
 **Аргументы**
 
 -   `dict_name` — имя словаря. [Строковый литерал](../syntax.md#syntax-string-literal).
--   `attr_names` — имя столбца словаря, [Строковый литерал](../syntax.md#syntax-string-literal), или кортеж [Tuple](../../sql-reference/data-types/tuple.md) таких имен.
--   `id_expr` — значение ключа словаря. [Выражение](../syntax.md#syntax-expressions), возвращающее значение типа [UInt64](../../sql-reference/functions/ext-dict-functions.md) или [Tuple](../../sql-reference/functions/ext-dict-functions.md), в зависимости от конфигурации словаря.
+-   `attr_names` — имя столбца словаря. [Строковый литерал](../syntax.md#syntax-string-literal), или кортеж [Tuple](../../sql-reference/data-types/tuple.md) таких имен.
+-   `id_expr` — значение ключа словаря.  [Expression](../../sql-reference/syntax.md#syntax-expressions) возвращает пару "ключ-значение" словаря или [Tuple](../../sql-reference/functions/ext-dict-functions.md), в зависимости от конфигурации словаря.
```
Comment (mathalex) [inline docs/ru/sql-reference/functions/ext-dict-functions.md:27]:
```suggestion
-   `id_expr` — значение ключа словаря. [Expression](../../sql-reference/syntax.md#syntax-expressions) возвращает пару "ключ-значение" словаря или [Tuple](../../sql-reference/functions/ext-dict-functions.md), в зависимости от конфигурации словаря.
```

File: docs/ru/sql-reference/functions/ext-dict-functions.md [hunk 6]
```diff
@@ -237,7 +237,7 @@ dictHas('dict_name', id)
 **Аргументы**
 
 -   `dict_name` — имя словаря. [Строковый литерал](../syntax.md#syntax-string-literal).
--   `id_expr` — значение ключа словаря. [Выражение](../syntax.md#syntax-expressions), возвращающее значение типа [UInt64](../../sql-reference/functions/ext-dict-functions.md) или [Tuple](../../sql-reference/functions/ext-dict-functions.md) в зависимости от конфигурации словаря.
+-   `id_expr` — значение ключа словаря.  [Expression](../../sql-reference/syntax.md#syntax-expressions) возвращает пару "ключ-значение" словаря или [Tuple](../../sql-reference/functions/ext-dict-functions.md) в зависимости от конфигурации словаря.
```
Comment (mathalex) [inline docs/ru/sql-reference/functions/ext-dict-functions.md:240]:
```suggestion
-   `id_expr` — значение ключа словаря. [Expression](../../sql-reference/syntax.md#syntax-expressions) возвращает пару "ключ-значение" словаря или [Tuple](../../sql-reference/functions/ext-dict-functions.md) в зависимости от конфигурации словаря.
```

**Review by mathalex — APPROVED** [review]

**Pull request merged**